src/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy) {
+  constructor(private readonly configService: ConfigService) {
+    const isProd = configService.get<string>('NODE_ENV') === 'production';
```

Comment:
```suggestion
    const IS_PROD = configService.get<string>('NODE_ENV') === 'production';
```
값이 변경되지 않는 상수는 일반적으로 UPPER_SNAKE_CASE 로 선언합니다